### PR TITLE
Re-added old drag & drop functionality

### PR DESCRIPTION
**Pull request by rahulvaddepalli2**
Re-added old drag & drop functionality to MainPanel
- Preserved toolTile's drag & drop functionality

### CODE REVIEW

File: src/main/kotlin/io/github/inductiveautomation/kindling/internal/FileTransferHandler.kt [hunk 1]
```diff
@@ -7,7 +7,7 @@ import java.io.IOException
 import javax.swing.TransferHandler
 
 class FileTransferHandler(
-    private val predicate: (File) -> Boolean,
+    private val predicate: (File) -> Boolean = { true },
```
Comment (paul-griffith) [inline src/main/kotlin/io/github/inductiveautomation/kindling/internal/FileTransferHandler.kt:10]: Can you explain why this change was necessary?

Comment (rahulvaddepalli2) [reply src/main/kotlin/io/github/inductiveautomation/kindling/internal/FileTransferHandler.kt:10]: Since there was no predicate before the tile change was made, this mimics the old behavior of FileTransferHandler before it had a predicate.

If preferred, I can remove the default value & pass it in manually.

Comment (paul-griffith) [reply src/main/kotlin/io/github/inductiveautomation/kindling/internal/FileTransferHandler.kt:10]: Ah, got it, now I'm following. No, this is fine.

**Review by paul-griffith — APPROVED** [review]
